velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -336,6 +336,9 @@ class ExpressionFuzzer {
         unsupportedFunctionSignatures,
         (double)unsupportedFunctionSignatures / totalFunctionSignatures * 100);
 
+    // Add additional signatures that are not in function registry.
```

Comment:
nit: Could we make the comment more specific that we're adding signatures for "and" and "or"?

Reply:
Hey sorry just saw your comments. I added this somewhat generic comment because I expect more special form expressions to be added in the future, like if/switch and such. This way we don't need to add comments to every call. And in fact we might want to just merge them all into a single helper method.